pgmpy/tests/test_models/test_LinearGaussianBayesianNetwork.py

```diff
@@ -195,5 +195,15 @@ def test_get_random_cpds(self):
         cpds = model_lin.get_random_cpds()
         self.assertEqual(len(cpds), len(model.nodes()))
 
+    def test_get_random(self):
+        model = LinearGaussianBayesianNetwork.get_random(
+            n_nodes=5, edge_prob=0.5, latents=False
+        )
+        self.assertIsInstance(
```

Comment:
Could you add further tests for the get_random method here? For example, testing some of the other arguments, number of nodes in the generated model, node names, etc.